lisa/testsuite.py

```diff
@@ -630,7 +631,10 @@ def start(
                 constants.RUN_LOCAL_LOG_PATH
             ).as_posix()
             case_result.set_status(TestStatus.RUNNING, "")
-            case_timeout = case_result.runtime_data.metadata.timeout
+            case_timeout = (
```

Comment:
When is this change needed? If it needs to extend the timeout, it should take max of both. If it wants to run test cases faster, it should take min of them. They are conflict goals.

For the `or` condition, it takes case_runbook higher priority, for some designed long running test cases, they will mostly failed. It's not right to overwrite case settings always.

Reply:
Changed it, it will take the max of both.
However, it will consider the one in metadata if the one passed from outside is None